python/cuml/svm/svc.pyx

```diff
@@ -241,6 +270,7 @@ class SVC(SVMBase):
 
         self._unpack_model()
         self.fit_status_ = 0
+        self.fit_once_ = 1
```

Comment:
Can fit_status be the flag we use to understand whether a fit is done? I hate to add another flag to manage

Reply:
I think that should be possible; `fit_status_` was only initialized within `fit` and it is used to indicate if `fit` was successful or not; so, I thought it might be better to have a separate flag to indicate that `fit` was not called at all.